docs/MIGRATING_TO_1_0.md

```diff
@@ -0,0 +1,29 @@
+# Migrating from 0.3 to 1.0
+
+## Chained operations
+[`chain()`](https://zenika.github.io/immutadot/immutadot/0.3/seq.html#.chain) has been removed in favor of [`flow()`](https://zenika.github.io/immutadot/immutadot/1.0/core.html#.flow).
```

Comment:
I think parenthesis are unnecessary

Reply:
Which parenthesis ?

Reply:
Which parenthesis ?

Reply:
after functions name

Reply:
I find it more clear, I'd like to keep these at least the first time I mention a function in the section

Reply:
I prefer using function keyword directly rather than parenthesis. But it's ok for me 👍